components/brave_ads/browser/notification_helper_win.cc

```diff
@@ -65,38 +65,6 @@ NotificationHelperWin::NotificationHelperWin() = default;
 NotificationHelperWin::~NotificationHelperWin() = default;
 
 bool NotificationHelperWin::ShouldShowNotifications() {
-  if (IsFullScreenMode()) {
-    LOG(WARNING) << "Notification not made: Full screen mode";
-    return false;
-  }
```

Comment:
This business logic is still required as ads should not be shown when the app is full-screen